verifiers/internal/gha/provenance.go

```diff
@@ -297,7 +321,25 @@ func VerifyProvenance(env *dsselib.Envelope, provenanceOpts *options.ProvenanceO
 		if err := isValidDelegatorBuilderID(prov); err != nil {
 			return err
 		}
-		// Note: `provenanceOpts.ExpectedBuilderID` is provided by the user.
+
+		// If ExpectedID is empty, check to see if it is a trusted builder.
+		// If empty and trusted builder, populate with path from provenance,
```

Comment:
this comment is off `If empty and trusted builder, populate with path from provenance`. If empty, we expect a trusted builder

Reply:
Right, I guess what I was trying to say was

`// If empty, then a trusted builder is expected, to populate provenanceOpts.ExpectedBuilderID with that builder`